examples/with-redux-toolkit/pages/_app.js

```diff
@@ -1,13 +1,14 @@
-import React from 'react'
-import { Provider } from 'react-redux'
-import { store } from '../store'
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import store from '../store';
```

Comment:
I believe the coding style suggest to have no semi-colon (see `.prettierrc`)?

Reply:
By personal experience I prefer to use semi-colon to avoid unwanted issues, by example

```javascript
const hello = 'Hello World'
(() => {
  console.log(hello)
})()
```

The snippet fails just because the missing semi-colon

Reply:
It would [fail](https://github.com/zeit/next.js/pull/12858/checks?check_run_id=674860196) the linting though.